third_party/openmetadata/prepare_OM_for_fybrik.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2020 IBM Corp.
```

Comment:
why do you need this start connector here ?

Reply:
The NewOpenMetadataAPIService() method runs a prepareOpenMetadataForFybrik() method which creates the custom properties that we need.

Reply:
I removed the go code.
The preparation is now done using the same executable as that of the openmetadata-connector (with different parameters).